src/cg.rs

```diff
@@ -18,11 +18,13 @@ pub struct CGImpl {
 
 impl CGImpl {
     pub unsafe fn new(handle: AppKitWindowHandle) -> Result<Self, SwBufError> {
+        let window = handle.ns_window as id;
         let view = handle.ns_view as id;
         let layer = CALayer::new();
-        let subview: id = NSView::alloc(nil).initWithFrame_(view.frame());
+        let subview: id = NSView::alloc(nil).initWithFrame_(NSView::frame(view));
         layer.set_contents_gravity(ContentsGravity::TopLeft);
         layer.set_needs_display_on_bounds_change(false);
+        layer.set_contents_scale(window.backingScaleFactor());
```

Comment:
I'm unsure if this will always be correct, e.g. if the view is moved to another monitor?

Reply:
Yes, this won't update automatically if the window's backing scale factor changes dynamically :(. I haven't looked into doing that, but agree that it would make sense to do, generally. I'd prefer it separate from this PR, as the lack of any scaling right now is pretty bad as-is :)